wpimath/src/main/native/include/frc/estimator/PoseEstimator.inc

```diff
@@ -53,6 +56,26 @@ void PoseEstimator<WheelSpeeds, WheelPositions>::ResetPosition(
   m_poseEstimate = m_odometry.GetPose();
 }
 
+template <typename WheelSpeeds, typename WheelPositions>
+void PoseEstimator<WheelSpeeds, WheelPositions>::ResetPose(const Pose2d& pose) {
+  m_odometry.ResetPose(pose);
+  m_odometryPoseBuffer.Clear();
+}
+
+template <typename WheelSpeeds, typename WheelPositions>
+void PoseEstimator<WheelSpeeds, WheelPositions>::ResetTranslation(
+    const Translation2d& translation) {
+  m_odometry.ResetTranslation(translation);
+  m_odometryPoseBuffer.Clear();
+}
+
+template <typename WheelSpeeds, typename WheelPositions>
+void PoseEstimator<WheelSpeeds, WheelPositions>::ResetRotation(
+    const Rotation2d& rotation) {
+  m_odometry.ResetTranslation(rotation);
```

Comment:
This should be ResetRotation? It would be good if some test was added that instantiated these templates to catch compile errors like this.